packages/react/toast/src/Toast.tsx

```diff
@@ -805,18 +809,43 @@ const ToastAnnounceExclude = React.forwardRef<
   ToastAnnounceExcludeElement,
   ToastAnnounceExcludeProps
 >((props: ScopedProps<ToastAnnounceExcludeProps>, forwardedRef) => {
-  const { __scopeToast, altText, ...announceHiddenProps } = props;
+  const { __scopeToast, altText, ...announceExcludeProps } = props;
 
   return (
     <Primitive.div
-      data-radix-toast-announce-exclude={altText ? undefined : ''}
+      data-radix-toast-announce-exclude=""
```

Comment:
nit:
```suggestion
      data-radix-toast-announce-exclude
```

Reply:
This will output `data-radix-toast-announce-exclude="true"`, though I notice we tend to prefer empty string for the more compact `data-radix-toast-announce-exclude` in other data attributes.

I'm cool with the change but curious on the distinction here?

Reply:
Ah yeah good point, I forgot that.